FutureFeatures.md

```diff
@@ -561,3 +561,13 @@ static signature validation check. This could be improved by allowing:
   check of a call to a heterogeneous table;
 * any other specific GC reference type, effectively allowing WebAssembly code
   to implement a variety of rooting API schemes.
+
+### Memset and Memcpy Operators
+
+Copying and clearing large memory regions is very common. This can be done in
+the MVP via `i32.load` and `i32.store`, but this is not very fast. The
```

Comment:
I'm not sure the speed argument is well stated: a fast `mem*` is dependent on architecture, and the load/store approach requires more bytecode while forcing VMs to recognize the loop.

Overall the reasons to have `mem*` aren't huge, but they seem at least worth considering.

I'd also mention that some things are expected from dev-side compilers such as LLVM. I don't expect to see small `mem*` operations in WebAssembly.